doc/faq/README.md

```diff
@@ -0,0 +1,21 @@
+# FAQ
+
+## What is *latency* and *duration*?
+
+![](mongoose-lat-dur.png)
```

Comment:
I think it'd be better to implicitly describe the X axis via adding "t" (or "time") to it.